pay-api/src/pay_api/services/statement.py

```diff
@@ -163,3 +168,26 @@ def get_statement_report(statement_id: str, content_type: str, template_name='st
         current_app.logger.debug('>get_statement_report')
 
         return report_response, report_name
+
+    @staticmethod
+    def get_summary(auth_account_id: str):
+        """Get summary for statements by account id."""
+        # This is written outside of the model, because we have multiple model references that need to be included.
+        # If we include these references inside of a model, it runs the risk of having circular dependencies.
+        # It's easier to build out features if our models don't rely on other models.
+        result = db.session.query(func.sum(InvoiceModel.total - InvoiceModel.paid).label('total_due'),
+                                  func.min(InvoiceModel.overdue_date).label('oldest_overdue_date')) \
+            .join(PaymentAccountModel) \
+            .join(StatementInvoicesModel) \
```

Comment:
Uses Statement invoices, so this only shows invoices that have statements. Otherwise I would have moved this into the invoice service.